tasks/translations.md

```diff
@@ -0,0 +1,132 @@
+# GCI Tasks: Creating translation style guides
+
+## Prerequisites
+
+* Update your working copy of Zulip and then create a feature branch. [Learn
+  how](../before-every-task.md).
+
+* CLONE_GIT_REPO (for anything that doesn't require a working dev environment)
+
+## Background
+
+Zulip is a tool used by many teams daily. In order to make it more approachable
+to users, there is an ongoing translation effort. [Read more about translating
+Zulip](https://zulip.readthedocs.io/en/latest/translating.html).
+
+This group of tasks is about creating common translation style guides, to give
+guidance on how Zulip should be translated into specific languages. Currently
+there are guides for [Polish](https://zulip.readthedocs.io/en/latest/polish.html)
+and [Spanish](https://zulip.readthedocs.io/en/latest/spanish.html).
+
+## Task Descriptions
+
+Let *language* and `<language>` be the language mentioned in the task that
+brought you here.
+
+### Task Type A: Research web application translations
+
+* Find three different web applications that have been translated into the
+*language*. Some examples of applications translated into multiple languages:
+  * Facebook
+  * GMail
+  * Twitter
+  * Instagram
+  * Skype
+  * YouTube
+  * Wikpedia
+
+* Look for common words and phrases associated with web applications such as
+*log in*, *home*, *reply* etc. Take screenshots of the applications with the
+interface both in English and *language* - save the English screenshots as
+`<application>_EN.png` and the *language* screenshots as `<application>.png`.
+Add the screenshots to a new folder `translations/<language>/research`.
+
+* Think about the language used - is it more formal or informal? Are there any
+specific forms that should or shouldn't be used? What words or phrases are most
+commonly used? What could sound awkward?
+
+* Are there any other specific requirements for the *language*? Does the
+application interface change when you switch languages? Is the *language*
+right-to-left or left-to-right? Does it require any special encodings or
+characters?
+
+* Write your notes as `notes.md` in `translations/<language>/research`.
+
+* Create a commit with the screenshots and notes, with commit message
+`translations: Research translations for <language>.`.
+
+* Create a pull request in the `zulip/zulip-gci` repository, with title
+`translations: Research translations for <language>.`.
+
+*Completion criteria*: The mentors will verify that you successfully created a
+pull request with at least 6 screenshots of 3 different interfaces and notes.
+
+### Task Type B: Write general rules for translations
+
+* Create a new file `general_rules.md` in `translations/<language>`.
```

Comment:
I think we've been using general-rules.md rather than general_rules.md as our convention for documentation files.